cmake/modules/FindRif.cmake

```diff
@@ -35,8 +35,11 @@ find_library(RIF_LIBRARY
 if(WIN32)
     set(RIF_BINARIES
         ${RIF_LOCATION_LIB}/dxcompiler.dll
+	${RIF_LOCATION_LIB}/dxil.dll
         ${RIF_LOCATION_LIB}/MIOpen.dll
         ${RIF_LOCATION_LIB}/RadeonImageFilters.dll
+        ${RIF_LOCATION_LIB}/OpenImageDenoise.dll
```

Comment:
Why do we need it?

Reply:
I added everything RIF repo has

Reply:
At least RadeonML.dll is needed for sure. dxil.dll probably same

Reply:
Please remove this line, we don't use OIDN.

Reply:
What about for CPU + denoising?  Or at least we should discuss for future if that makes sense for final renders

Reply:
I only said that we don't use it at the moment so it makes no sense to add this 40 Mb dll to the package.

Surely we can use the OIDN if needed. It's more robust than the AI denoiser. Do you want me to add a way for users to select a denoiser filter and add support of the OIDN?

Reply:
What do we currently if CPU is enabled and denoising is turned on?  The RIF denoiser uses OIDN on cpu

Reply:
https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/pxr/imaging/plugin/hdRpr/rprApi.cpp#L2163

EAW denoiser is used in this case.

> The RIF denoiser uses OIDN on cpu

This is really surprising if it's true.

Reply:
Sorry @hshakula I should amend my comment "RIF AI Denoise" uses OIDN if run on CPU.

Reply:
What do you guys mean when you say RIF running on CPU? Do you able to create OpenCL context on CPU? For me, RIF never detected my CPU as a valid device.

Reply:
Let's keep it as is for now (no OIDN) and talk about this in a future task.